platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LogoutService.java

```diff
@@ -80,17 +80,14 @@ public Response getActionProviders(@Context HttpServletRequest request)
                             "",
                             true);
 
-                    if (displayName != null
```

Comment:
Did we still want the null check here? In which scenarios would the displayName be null?